styles/themes/teammember/_component-overrides.scss

```diff
@@ -1,3 +1,5 @@
+@import '../../typography';
```

Comment:
Actually, there's a reference to font-size-larger which is in typography. Should that import be done externally?